src/gatherers/user-scripts.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Gather = require('./gather');
+const fs = require('fs');
+const path = require('path');
+const scriptsDir = path.resolve(__dirname, '..', '..', 'user_scripts');
```

Comment:
someone installing lighthouse from npm is going to want to be able to specify regular paths, I think, and not rely on putting them in `user_scripts`